.github/workflows/github_actions_build.yml

```diff
@@ -40,3 +40,55 @@ jobs:
         cd ..\vt
         nmake CHTYPE_32=Y -f Makefile.vc demos
       shell: cmd
+
+  XbuildWindows:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { name: W32_VT,     arch: _w32, ENV: vt,     PREFIX: i686-w64-mingw32 }
+          - { name: W32_WINCON, arch: _w32, ENV: wincon, PREFIX: i686-w64-mingw32 }
+          - { name: W32_WINGUI, arch: _w32, ENV: wingui, PREFIX: i686-w64-mingw32 }
+          - { name: W32_SDL2,   arch: _w32, ENV: sdl2,   PREFIX: i686-w64-mingw32 }
+          - { name: W64_VT,     arch: _w64, ENV: vt,     PREFIX: x86_64-w64-mingw32 }
+          - { name: W64_WINCON, arch: _w64, ENV: wincon, PREFIX: x86_64-w64-mingw32 }
+          - { name: W64_WINGUI, arch: _w64, ENV: wingui, PREFIX: x86_64-w64-mingw32 }
+          - { name: W64_SDL2,   arch: _w64, ENV: sdl2,   PREFIX: x86_64-w64-mingw32 }
+          - { name: WoA_VT,     arch: _a64, ENV: vt,     PREFIX: aarch64-w64-mingw32 }
+          - { name: WoA_WINCON, arch: _a64, ENV: wincon, PREFIX: aarch64-w64-mingw32 }
+          - { name: WoA_WINGUI, arch: _a64, ENV: wingui, PREFIX: aarch64-w64-mingw32 }
+    steps:
+    - uses: actions/checkout@master
+    - name: ${{ matrix.name }}
+      shell: bash
+      run: |
+        ## download llvm-MinGW
+        assetsUrl=$(wget -qO- https://github.com/mstorsjo/llvm-mingw/releases/latest | grep "expanded_assets" | grep -Po 'https[^"]+')
```

Comment:
downloading each time seems too much, as far as I remember there's an option to store and re-use later

Reply:
It really doesn't matter, the source is in github, so it is as fast as using caches. Using caches is the method you mention, but it needs to download it no matter what as github store caches internally in .tar.xz format. So it won't be faster at all.

I am using that method for [nano for windows](https://github.com/okibcn/nano-for-windows) but for another reason. I use an optimized version of LLVM-MinGW there using musl under Alpine. So, I had to build my own tweaked version. I refresh the cache every week as it takes about 4 hours to build all the tools for all the environments.

Reply:
> It really doesn't matter, the source is in github, so it is as fast as using caches.

Just out of interest: Do you have any docs that the "storage on github" is identical between the release artifacts and the caches? The first is reasonable to be kept with a CDN, while it makes more sense to keep the cache "as near as possible" to its single using worker(s).

Reply:
Checking back again - using caches (which also allows to check its state and drop the cache if necessary) really seems to be preferable. Please modify and we can pull this in.